docs/_partners/lifecycle-planning.md

```diff
@@ -49,27 +49,25 @@ Learn how to use the Zapier Platform with our [visual builder Quick Start tutori
 
 ### Test Your Integration
 
-While you're building your integration, it's best to test authentication, triggers, and actions as you add them to your app. In Platform UI, you can test each step live in the editor as its built. With both Platform UI and CLI, have a separate tab open in your browser to the user-facing [Zap editor](https://zapier.com/app/editor), then refresh anytime you add a new trigger or action to test it live immediately.
+While you're building your integration, it's best to test authentication, triggers, and actions as you add them to your app. In the Platform UI, you can test each step live in the editor as it's built. With both Platform UI and CLI, have a separate tab open in your browser to the user-facing [Zap editor](https://zapier.com/app/editor), then refresh anytime you add a new trigger or action to test it live immediately.
 
 > **Note:** You may have to refresh a few times for new triggers and actions to appear.
 
-After you're done building, you will want to invite users to try your integration before making it available to a wider audience.
+After you're done building, invite users to try your integration before making it available to a wider audience.
 
 #### Invite Others to Help Test Your Integration
 
-![Invite people to use Zapier integration](https://cdn.zapier.com/storage/photos/4fb2802664a2fe72f44d223e892ca88e.png)
```

Comment:
This screenshot is out of date, but I wasn't sure how to add a new one properly, so I've just removed it for now.